### PR TITLE
remove duplicate declaration of 'input'

### PR DESCRIPTION
**Pull request by CBonade**
## Description
https://app.zenhub.com/workspaces/vsp-5cedc9cce6e3335dc5a49fc4/issues/department-of-veterans-affairs/va.gov-team/25660

This converts the Disability Rating Calculator test from Nightwatch to Cypress.  Prior helper functions have been moved to Cypress custom commands.  The test properly tests both correct inputs/results as well as validation when improper input is entered.

## Testing done

Tested with Cypress runner in Chrome.  Makes it through all 343 steps.

## Screenshots
image

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/disability-benefits/disability-rating-calculator/tests/00-all-fields.cypress.spec.js [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+  combinedRating: {
+    exact: 50,
+    rounded: 50,
+  },
+};
+Cypress.Commands.add('fillRatings', input => {
+  const { ratings } = input;
+  const ratingsLength = ratings.length;
+
+  for (let x = 0; x < ratingsLength; x++) {
```
Comment (ddzz) [inline src/applications/disability-benefits/disability-rating-calculator/tests/00-all-fields.cypress.spec.js:23]: Really minor point: can we use something other than `x` for the variable name here?

Comment (CBonade) [reply src/applications/disability-benefits/disability-rating-calculator/tests/00-all-fields.cypress.spec.js]: Replaced this with `ratingsIndex` to be more descriptive of it's purpose.

**Review by pjhill — APPROVED** [review]
LGTM

**Review by Mottie — APPROVED** [review]

**Comment by pjhill** [comment]
Resolves [25708](https://github.com/department-of-veterans-affairs/va.gov-team/issues/25708)

**Pull request merged**